tf2onnx/graph.py

```diff
@@ -141,17 +147,22 @@ def __repr__(self):
         return "<onnx op type='%s' name=%s>" % (self.type, self._op.name)
 
     def get_attr(self, name, default=None):
-        """Get attribute map."""
+        """Get raw attribute value."""
         attr = self.attr.get(name, default)
         return attr
 
     def get_attr_int(self, name):
-        """Get attribute map."""
-        attr = self.attr.get(name)
+        """Get attribute value as int."""
+        attr = self.get_attr(name)
         utils.make_sure(attr is not None, "attribute %s is None", name)
         attr = attr.i
         return attr
 
+    def get_attr_str(self, name, encoding="utf-8"):
+        """Get attribute value as string."""
+        attr = self.get_attr(name)
```

Comment:
shall we keep the same behavior of get_attr_int, e.g. fast fail. if the attr is None.

Reply:
for get_attr_int, a None attr value cannot be converted to int, so the check is required.
but for get_attr_str, return a None attr value means value does not exist, caller can decide what to do next, either fail or continue.

Reply:
get_attr() should be the function that check the attribute existence. when calling get_attr_str(), the dev is expecting the attribute exists and it's type is string in my option.